src/PlusImageProcessing/Testing/vtkPlusTransverseProcessEnhancerTest.cxx

```diff
@@ -36,10 +62,130 @@ int main(int argc, char **argv)
 
   vtkPlusLogger::Instance()->SetLogLevel(verboseLevel);
 
-  ///////////////
+  //check for various non-optional command line arguments
+  int commandCheckStatus = NULL;
+
+  if (inputFileName.empty())
+  {
+    std::cerr << "--input-seq-file not found!" << std::endl;
+    commandCheckStatus = EXIT_FAILURE;
+  }
+  if (outputFileName.empty())
+  {
+    std::cerr << "--output-seq-file not found!" << std::endl;
+    commandCheckStatus = EXIT_FAILURE;
+  }
+  if (configFileName.empty())
+  {
+    std::cerr << "--config-file not found!" << std::endl;
+    commandCheckStatus = EXIT_FAILURE;
+  }
+
+  if (commandCheckStatus == EXIT_FAILURE)
+  {
+    return EXIT_FAILURE;
+  }
 
-  // TODO : write code that tests the functionality of your tool here
   vtkSmartPointer<vtkPlusTransverseProcessEnhancer> enhancer = vtkSmartPointer<vtkPlusTransverseProcessEnhancer>::New();
 
+  // Read input sequence
+  vtkSmartPointer<vtkPlusTrackedFrameList> trackedFrameList = vtkSmartPointer<vtkPlusTrackedFrameList>::New();
+  if (trackedFrameList->ReadFromSequenceMetafile(inputFileName.c_str()) == PLUS_FAIL)
+  {
+    return EXIT_FAILURE;
+  }
+  vtkSmartPointer<vtkImageCast> castToUchar = vtkSmartPointer<vtkImageCast>::New();
+  castToUchar->SetOutputScalarTypeToUnsignedChar();
+
+  int numberOfFrames = trackedFrameList->GetNumberOfTrackedFrames();
+  std::cout << "Number of frames in input: " << numberOfFrames << std::endl;

```

Comment:
Instead of print to cout, use LOG_INFO (everywhere, not just here) to make sure everything is recorded in log files. Due to some reason we still print command help to cout (probably we should change that, for consistency)but otherwise we almost never print directly to cout.